.gitignore

```diff
@@ -3,7 +3,7 @@
 build/device/**/*.pyc
 epsilon.elf
 epsilon.map
-.vscode
+/.vscode/*
```

Comment:
Why ?

Reply:
little mistake

Reply:
should i commit on top of that ?

Reply:
I think you can, because it will be squashed at the end…

Reply:
Done.